README.md

```diff
@@ -97,6 +100,13 @@ You can put a manifest in the same folder with the data files, or you can put it
 - overwrite: if set to “true”, CLI will upload a data file to overwrite the data file with same name that already exists in the Data Hub target storage. If set to “false”, CLI will not upload a data file if a data file with the same name exists in the Data Hub target storage.
 - dryrun: if set to “true”, CLI will not upload any data files to the Data Hub target storage. If set to “false”, CLI will upload data files to the Data Hub target storage.
 
+### File ID generation
+File.tsv template downloaded from Data Model viewer doesn’t contain file Keys/IDs column, because the system will generate it for you. The generated final manifest will be saved in the same place the file manifest is and has a “-final” suffix . For example, file-final.tsv. 
```

Comment:
recommending the following changes to this section:
"
The file.tsv template, downloaded from Data Model viewer, doesn’t contain file Keys/IDs column because it will be generated for you by the system. The generated final manifest will be saved in the same place as the file manifest and a “-final” suffix with be added . For example, file-final.tsv.
"